### PR TITLE
MET-5056: Convert to spring boot rest application

### PR DESCRIPTION
**Pull request by stzanakis**

### CODE REVIEW

File: clio-linkchecking/src/main/resources/application.properties.example [hunk 1]
```diff
@@ -1,3 +1,11 @@
+#Spring
+#logging.config=/data/logging/log4j2.xml
+logging.config=log4j2.xml
+spring.autoconfigure.exclude=\
+  org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration, \
+  org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration, \
+  org.springframework.boot.autoconfigure.data.mongo.MongoDataAutoConfiguration
+
```
Comment (jeortizquan) [inline clio-linkchecking/src/main/resources/application.properties.example:8]: is this needed also? ```org.springframework.boot.autoconfigure.solr.SolrAutoConfiguration```

**Review by jeortizquan — COMMENTED** [review]
Nice resturcture, I just have this two observations.

File: pom.xml [hunk 2]
```diff
@@ -27,13 +26,18 @@
...
 		<version.hibernate>5.4.10.Final</version.hibernate>
+		<version.postgresql>42.4.1</version.postgresql>
+		<version.spring.boot>2.7.8</version.spring.boot>
 		<version.spring>5.2.3.RELEASE</version.spring>
+		<version.springdoc.openapi>1.7.0</version.springdoc.openapi>
+		<version.elastic.apm>1.34.1</version.elastic.apm>
 		<version.slf4j>1.7.30</version.slf4j>
 		<version.log4j>2.17.1</version.log4j>
+		<version.junit>5.7.2</version.junit>
 		<version.swagger>3.0.0</version.swagger>
```
[the remark this answers is not in the document]
Comment (stzanakis) [reply pom.xml:41]: Indeed! Removed.

File: clio-linkchecking/src/main/resources/application.properties.example (hunk 1 above)
Comment (stzanakis) [reply clio-linkchecking/src/main/resources/application.properties.example:8]: I don't think so, but I will add it just in case for now.

**Pull request merged**